src/Features/Context/TiqrContext.php

```diff
@@ -242,25 +222,23 @@ public function appAuthenticates(
             'notificationAddress' => $notificationAddress,
         ];
         // Internal request does not like an absolute path.
-        $authenticationUrl = str_replace('https://tiqr.stepup.example.com', '', $authenticationUrl);
+        $authenticationUrl = str_replace('https://tiqr.dev.openconext.local', '', (string) $authenticationUrl);
```

Comment:
Fixed url (maybe use params?)

Reply:
Again, testcode, not too concerned for having this fully configurable.